app/scripts/lib/createDupeReqFilterMiddleware.test.js

```diff
@@ -20,11 +20,11 @@ describe('createDupeReqFilterMiddleware', () => {
     const endMock = jest.fn();
 
     filterFn(request, undefined, nextMock, endMock);
-    expect(nextMock).toBeCalledTimes(1);
-    expect(endMock).not.toBeCalled();
+    expect(nextMock).toHaveBeenCalledTimes(1);
+    expect(endMock).not.toHaveBeenCalled();
 
     filterFn(request, undefined, nextMock, endMock);
-    expect(nextMock).toBeCalledTimes(1);
-    expect(endMock).toBeCalledTimes(1);
```

Comment:
These changes were introduced from the yarn lint:fix command. I can take them out if necessary.

Reply:
Hmm. That shouldn't happen, but I did confirm this to be the case just because I was curious. Weird.

This problem does go away after rebasing/merging latest develop though.

Reply:
okay, thanks for the insight @Gudahtt, I still need to get familiar with the pipeline